_posts/2021-10-26-g2p-preprocess.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Tutorial: Solve inconsistencies in your text with a g2p pre-processing mapping"
+author: AidanPine
+categories: []
+tags: [ tutorial, explanation, tech-showcase, g2p ]
+description: "Tutorial: Solve inconsistencies in your text with a g2p pre-processing mapping"
+featured: false
+hidden: false
+---
+
+This is the 6th blog post in a seven-part series about a software tool called `g2p`. In this post we'll discuss how to use `g2p` to do the common natural language processing task of text normalization.
+
+### G2P Blog Series Index
+
+- [Background]({{ "g2p-background" | absolute_url }})
+- [How to write a basic mapping in G2P Studio]({{ "g2p-basic-mappings-gui" | absolute_url }})
+- [Writing mappings on your computer]({{ "g2p-basic-mappings-local" | absolute_url }})
+- [Advanced mappings]({{ "g2p-advanced-mappings" | absolute_url }})
+- [ReadAlong Studio & Other Applications]({{ "g2p-applications" | absolute_url }})
+- Preprocessing mappings
+- [Contributing]({{ "g2p-contributing" | absolute_url }})
+
+## Adding a 'pre-processing' mapping
+
+It's often not sufficient to just write a mapping between the characters in a language's orthography and the IPA. As seen in [use case #2](#use-case-2-a-language-with-multiple-writing-systems) and [use case #3](#use-case-3-converting-from-legacy-writing-systems). Real-world text input is pretty messy, and if we want ReadAlongs or Convertextract - or any other tool that uses `g2p` - to work properly, we need to account for as much of that messiness as possible. Generally speaking, solving this kind of messiness is usually called 'Text Normalization'.[^n] This 'normalization' can either be about ensuring that the same Unicode characters are used consistently, or it can also be about converting symbols into their pronounced form, like & or 123.
```

Comment:
I'd change the second sentence to "As illustrated in use case 2 and use case 3 below", so the reader knows right away, without having to click, that they're about to read them and aren't expected to remember them from before.

 And maybe join it with the previous sentence since it's not a complete sentence.